index.html

```diff
@@ -1,220 +1,218 @@
+                        no accounts <br>  or sign-ups required.</p>
+                        <ul class="desi infolist">
+                            <li class="liststyle desi span">Html</li> 
+                           <li class="liststyle desi span">Css</li> 
+                             <li class="liststyle desi span">Javascript</li> 
+                        </ul>
+                        <button class="btn" >See Project</button>
+                </div>
             </div>
-            <div class="normal-button">
-              <button type="button" class="link-buttons">See Project</button>
-            </div>
-          </div>
-        </div>
-
-        <div class="work-container">
-          <div class="cards-work">
-            <div class="snapshoot"><img src="./images/Snapshoot_Portfolio(1).png" alt=""></div>
-            <div class="primary-text">
-              <h2>Multi-Post <br>Stories</h2>
-              <img src="./images/Frame2.png" alt="">
-              <p>A daily selection of privately <br>personalized reads; no accounts or <br>sign-ups required.</p>
-            </div>
-            <div>
-              <ul class="lang">
-                <li>html</li>
-                <li>css</li>
-                <li>javaScript</li>
-              </ul>
-            </div>
-            <div class="normal-button">
-              <button type="button" class="link-buttons">See Project</button>
-            </div>
-          </div>
-        </div>
-
-        <div class="work-container">
-          <div class="cards-work">
-            <div class="snapshoot"><img src="./images/Snapshoot_Portfolio(2).png" alt=""></div>
-            <div class="primary-text">
-              <h2>Tonic</h2>
-              <img src="./images/Frame2.png" alt="">
-              <p>A daily selection of privately <br>personalized reads; no accounts or <br>sign-ups required.</p>
+            <div  class="card">
+                <img class="grid-card"  src="images/Snapshoot2Portfolio.png"  alt="SecondProjectScreenShot">
+                <div class="TextArea">
+                    <h3>Multi-Post
+                       <br> Stories</h3>
+                    <ul class="infolist">
+                        <li class="desi font ">CANOPY</li>
+                        <li class="Bulledesi">Back End Dev</li>
+                        <li class="Bulledesi">2015</li>
+                    </ul>
+                    <p>Experimental content creation feature that allows users to add to an existing story over<br>
+                         the course of a day without spamming their friends.</p>
+                        <ul class="desi infolist">
+                            <li class="liststyle desi span">Html</li> 
+                            <li class="liststyle desi span">Css</li> 
+                            <li class="liststyle desi span">Javascript</li> 
+                        </ul>
+                        <button class="btn" >See Project</button>
+                </div>
```

Comment:
- [ ] Kindly make sure that the project cards display properly on width `768px` to match the Figma design. 

| Your design | Figma design |
| --- | --- |
| ![image](https://user-images.githubusercontent.com/62290545/161294832-06308c45-40fd-42bf-a50c-dd7e37d25a0d.png) | ![image](https://user-images.githubusercontent.com/62290545/161294899-cbdd229b-ae1e-457f-be00-04a937494a2c.png) |